src/foam/java/PropertyInfo.js

```diff
@@ -106,6 +106,12 @@ foam.CLASS({
             visibility: 'public',
             type: 'boolean',
             body: 'return ' + Boolean(this.propRequired) + ';'
+          },
+          {
+            name: 'of',
+            visibility: 'public',
+            type: 'String',
```

Comment:
This wouldn't return the `of` field that would be filled by the specific properties if it were ClassInfo.